webots_ros2_core/webots_ros2_core/differential_drive_node.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 1996-2020 Cyberbotics Ltd.
```

Comment:
It is probably worth mentioning this new class in this changelog: https://github.com/cyberbotics/webots_ros2/blob/0a291134bd626c49a1ded4911163ca3e8c5705f7/webots_ros2_core/CHANGELOG.rst